include/reduction_manager.h

```diff
@@ -36,9 +36,9 @@ namespace detail {
 				acc = m_op(acc, *static_cast<const DataT*>(data.get_pointer()));
 			}
 
-			auto host_buf = runtime::get_instance().get_buffer_manager().get_host_buffer<DataT, Dims>(
+			auto info = runtime::get_instance().get_buffer_manager().access_host_buffer(
```

Comment:
Can be const.

Reply:
I wonder why I made these all non-const 🤔 Fixed!